examples/pede/pede.md

```diff
@@ -0,0 +1,58 @@
+pede minimizer {#pede}
+==============
+This example uses pede_job.py to optimize the alignment parameters. The inputs are recursively searched for `*.bin` files that we assume were prepared by hps-java during the tracking step.
+These data files are then used to run the minimization scheme within pede.
+Finally, the results form pede are applied to the detector description file (`compact.xml`), generating a new iteration of the detector.
+
+# Parameters
+There are several special parameters that are separated depending on if they are required and how often they change between jobs.
+
+#### Requried Config
+These parameters are required but do not change often so they are put into a config file (e.g. `.hpsmc` in this directory) rather than the job JSON.
+
+| param              | description                                                              |
+|--------------------|--------------------------------------------------------------------------|
+| param\_map         | map of SVT sensors listing names, ID numbers, and other location details |
+| pede\_minimization | pede minimization settings to append to its steering file                |
+| java\_dir          | root directory of hps-java for applying pede results                     |
+
+
+#### Required Job
+These parameters are required and change often depending on the iteration and what alignment you are studying, so they are in the job JSON.
+
+| param         | description                                                                 |
+|---------------|-----------------------------------------------------------------------------|
+| inputs        | list of `bin` files or directories in which to recursively search for them  |
+| output\_files | mapping the pede output files to their destination names                    |
+| to\_float     | list of pede parameters to allow to float during minimization               |
+| detector      | name of detector that tracking was run on                                   |
```

Comment:
More detail on these parameters, specifically on how they are interpreted by the components.